src/epics/wiki.js

```diff
@@ -16,35 +17,46 @@ const fetchAllWikis = ( action$ ) => (
 				.map( ( ajaxResponse ) => {
 					const wikis = Object.keys( ajaxResponse.response.sitematrix )
 						.filter( ( key ) => {
-							return ( !isNaN( parseFloat( key ) ) && isFinite( key ) ) || key === 'specials';
+							return ( !isNaN( parseFloat( key ) ) && isFinite( key ) );
 						} )
 						.map( ( key ) => {
-							// Special is different for some reason.
-							if ( key === 'specials' ) {
-								return {
-									site: ajaxResponse.response.sitematrix[ key ]
-								};
-							}
-
 							return ajaxResponse.response.sitematrix[ key ];
 						} )
 						.reduce( ( state, data ) => (
 							[
 								...state,
 								...data.site.map( ( item ) => (
-									new Wiki( {
+									{
 										id: item.dbname,
-										domain: new URL( item.url ).hostname
-									} )
+										domain: new URL( item.url ).hostname,
+										family: item.code,
+										code: data.code
+									}
 								) )
 							]
-						), [] )
-						.reduce( ( state, data ) => ( state.set( data.id, data ) ), new Map() )
-						.sort( ( a, b ) => a.domain > b.domain );
+						), specialWikisList )
+						.sort( ( a, b ) => {
+							if ( a.code === b.code ) {
+								if ( a.family === 'wiktionary' && b.family !== 'wiki' ) {
```

Comment:
This looks like it will always sort wiktionaries first, but what about wikipedias? Shouldn't those also be sorted above wikibooks and wiktionaries? If we follow the same order as pageviews, we would want: wikipedia, wiktionary, wikibooks, wikinews, wikiquote, wikisource, wikiversity, wikivoyage. Would also be good to add a comment here explaining what we're doing.

Reply:
Everything gets sorted alphabetically. The only exceptions are wiktionary, which I bubble up unless the element on top is a Wikipedia, resulting in Wikipedia, Wiktionary, Wiki(all the others) alphabetically.

Wikipedias get sorted as firsts because their `code/family` is wiki, which just happens to be the first in order of all the types of wikis we got.

Reply:
Ah, I forgot that 'wiki' means Wikipedia! I was thinking wiki was just for the special wikis. Just proves that it needs a good explanation in a comment :)

Reply:
@kaldari I'm adding some comments to the sort function to make it more clear. Let me know if there is anything else I should change.